includes/dbFacile.php

```diff
@@ -210,8 +210,9 @@ function dbUpdate($data, $table, $where = null, $parameters = [])
     }
 
     try {
-        $result = Eloquent::DB()->update($sql, (array)$data);
-
+        $result = Eloquent::DB()->transaction(function () {
```

Comment:
Missing `use($data, $sql)`

Reply:
Do I just need to add that line in?
$result = Eloquent::DB()->transaction(function () {
    use($data, $sql)
Or does it need to be in function() or after that call? (apologies my php is not strong)

Reply:
I believe i've updated this correctly, but now it's working on PHP 7.2 and 7.4.  But failing on PHP 7.3.
Before the change it was the other way around.

Reply:
Fixed, thanks! that sorted the travis bulds.